src/python/pants/notes/2.5.x.md

```diff
@@ -2,6 +2,28 @@
 
 See https://www.pantsbuild.org/v2.5/docs/release-notes-2-5 for an overview of the changes in this release series.
 
+## 2.5.0.dev2 (Apr 16, 2021)
+
+### New Features
+
+* Handle URL requirements with constraints files. ([#11907](https://github.com/pantsbuild/pants/pull/11907))
```

Comment:
Bug fix?

Reply:
Disagree. This isn't something we advertised as ever working.

Reply:
That's fine w/ me, thus the question mark. Thanks